_includes/layouts/base.njk

```diff
@@ -84,7 +84,7 @@
           {{ content | safe }}
         {% endblock %}
 
-        {% if page.url.indexOf("/posts/") > -1 %}
+        {% if page.url and page.url.indexOf("/posts/") > -1 %}
```

Comment:
Can this ever be null-ish in 11ty?

Ah, is that caused by the `permalink: false`?

Reply:
That's right, I bumped into this issue when trying to add a `draft` feature by setting `permalink` to false and `excludeFromCollections` to true. And here is the [11ty permalink document](https://www.11ty.dev/docs/permalinks/#permalink-false) saying that `url` will be false if `permalink` is false.


Forgot to post the error message, here it is
```
Unhandled rejection in promise ([object Promise]): (more in DEBUG output)
> Unable to call `page["url"]["indexOf"]`, which is undefined or falsey
```